cartography/cli.py

```diff
@@ -65,6 +65,25 @@ def _build_parser(self):
             action="store_true",
             help="Restrict cartography logging to warnings and errors only.",
         )
+        # Optional export-only sink (off by default)
+        parser.add_argument(
+            "--export-file",
```

Comment:
Let's have a more specific name since (from my other comment) I think we will have at least this format and one more export format.